Neos.FluidAdaptor/Classes/Core/Widget/AbstractWidgetViewHelper.php

```diff
@@ -241,17 +241,17 @@ protected function initiateSubRequest()
                 $subResponse = $this->controller->processRequest($subRequest);
 
                 // We need to make sure to not merge content up into the parent ActionResponse because that _could_ break the parent response.
-                $content = $subResponse->getContent();
-                $subResponse->setContent('');
+                $content = $subResponse->getBody()->getContents();
+                $subResponse = $subResponse->withBody(Utils::streamFor(''));
             } catch (StopActionException $exception) {
                 $subResponse = $exception->response;
-                $subResponse->mergeIntoParentResponse($this->controllerContext->getResponse());
+                $this->controllerContext->getResponse()->replaceHttpResponse($subResponse);
                 throw $exception;
             } catch (ForwardException $exception) {
                 $subRequest = $exception->nextRequest;
                 continue;
             }
-            $subResponse->mergeIntoParentResponse($this->controllerContext->getResponse());
+            $this->controllerContext->getResponse()->replaceHttpResponse($subResponse);
```

Comment:
possibly different behaviour, as the method is unsafe: https://github.com/neos/flow-development-collection/issues/2492